README.md

```diff
@@ -1,27 +1,39 @@
 
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Portfolio-setup-and-mobile-version-skeleton
+# Programming languages
 
-> This is about using figma to built our mobile project.
+> This is about programming languages.
+
+- Desktop Version
+![screenshot](./screen-shoots/DesktopScreenS.png)
+
+- Mobile Version
+
+![screenshot](./screen-shoots/mobileScreenS.png)
+![screenshot](./screen-shoots/mobileSSbottom.png)
+![screenshot](./screen-shoots/speakers.png)
+![screenshot](./screen-shoots/Spekaers2.png)
 
-![screenshot](./SS/app-screenshot.png)
-![screenshot](./SS/about-screenshot.png)
-![screenshot](./SS/icons-screenshot.png)
-![screenshot](./SS/work-section.png)
-![screenshot](./SS/formscreenshot.png)
 
 Mobile ScreenShots
 
 ## Link of Web Page
 
-- GitHub Pages: [@eri8-9](https://eri8-9.github.io/Portfolio-setup-and-mobile-version-skeleton/
-)
+- GitHub Pages: [@eri8-9](https://eri8-9.github.io/first-capston/)
 
 ## Built With
 
 - HTML, CSS & JS
 
+## Live Demo
+
+[Live Demo Link](https://www.loom.com/share/bce69d74380141f996e1acff12aca0d1)
```

Comment:
- [ ] Please change the live demo link to the project link not the loom presentation video 😊

Reply:
https://eri8-9.github.io/first-capston/

- This is the link that should be on the Live Demo Link, as the name said, is the live demonstration, not the video. Please, add this link to the live demo section. 🐶